depends/check-psl1ght.sh

```diff
@@ -5,7 +5,7 @@
 if test ! $PSL1GHT; then { echo "ERROR: Set \$PSL1GHT before continuing."; exit 1; } fi
 
 ## Check for the $PSL1GHT directory.
-( ls -ld $PSL1GHT || mkdir -p $PSL1GHT ) 1>/dev/null 2>&1 || { echo "ERROR: Create $PSL1GHT before continuing."; exit 1; }
+#( ls -ld $PSL1GHT || mkdir -p $PSL1GHT ) 1>/dev/null 2>&1 || { echo "ERROR: Create $PSL1GHT before continuing."; exit 1; }
```

Comment:
Don't remove the sanity check.  Removing the `|| mkdir -p $PSL1GHT` is ok since the README now says you have to create it yourself.
(And if we are changing this anyway I'd prefer `test -d` over `ls -ld` for the check.  :smile:)